.github/ISSUE_TEMPLATE.md

```diff
@@ -10,83 +10,88 @@ If this is a BUG REPORT, please:
 If this is a FEATURE REQUEST, please:
   - Describe *in detail* the feature/behavior/change you'd like to see. Note,
     for simple and tiny features or change requests we encourage you opening
-    a pull request with a proposed implementaion.
+    a pull request with a proposed implementation.
 
 In both cases, be ready for followup questions, and please respond in a timely
 manner.  If we can't reproduce a bug or think a feature already exists, we
 might close your issue.  If we're wrong, PLEASE feel free to reopen it and
 explain why.
 -->
 
-# BUG REQUEST INFO:
+# BUG REQUEST INFO
```

Comment:
once you're fixing this line, could you please fix the "bug request" to become "but report"? :)